app/indexers/ephemera_box_indexer.rb

```diff
@@ -11,7 +11,8 @@ def to_solr
     {
       "title_tesim" => [decorated.title],
       "title_ssim" => [decorated.title],
-      "title_tsim" => [decorated.title]
+      "title_tsim" => [decorated.title],
+      "barcode_ssim" => [decorated.barcode]
```

Comment:
I think this is indexed already.

Reply:
It was originally failing without it, I will add another commit.